includes/init.php

```diff
@@ -347,9 +363,17 @@ function print_trailer( $include_nav_links = true, $closeDb = true,
   }
 
   return $ret . '
-<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->
-' // Adds an easy link to validate the pages.
-  . ( $DEMO_MODE == 'Y' ? '
+<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->' .
+    ( $ALLOW_HTML_DESCRIPTION !== 'Y' ? '' :
+    /* Your choices here are "basic", "standard" or "full". */ '
+    <script src="//cdn.ckeditor.com/4.6.0/basic/ckeditor.js"></script>
```

Comment:
Any reason to prefer 4.6.0 over 4.7.1?  This will be easier for most users.  But, it might be good to also check the local file system to see if it has been installed locally.  I'd still like the possibility of running everything local (no remote requests).